### ISSUE TITLE
StatisticsError: variance requires at least two data points

### ISSUE THREAD

**Issue by ma-zal** [dgomes/rrd_recorder#1]
During Home Assistant startup I see error in log:

```
homeassistant    | Traceback (most recent call last):
homeassistant    |   File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
homeassistant    |     result = self.fn(*self.args, **self.kwargs)
homeassistant    |   File "/config/custom_components/rrd/__init__.py", line 169, in rrd_update
homeassistant    |     statistics.stdev(entities_last_changed) <= conf[CONF_TOLERANCE]
homeassistant    |   File "/usr/local/lib/python3.8/statistics.py", line 797, in stdev
homeassistant    |     var = variance(data, xbar)
homeassistant    |   File "/usr/local/lib/python3.8/statistics.py", line 739, in variance
homeassistant    |     raise StatisticsError('variance requires at least two data points')
homeassistant    | statistics.StatisticsError: variance requires at least two data points
homeassistant    | 2020-09-01 09:33:53 ERROR (MainThread) [homeassistant] Error doing job: Future exception was never retrieved
homeassistant    | Traceback (most recent call last):
homeassistant    |   File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
homeassistant    |     result = self.fn(*self.args, **self.kwargs)
homeassistant    |   File "/config/custom_components/rrd/__init__.py", line 169, in rrd_update
homeassistant    |     statistics.stdev(entities_last_changed) <= conf[CONF_TOLERANCE]
homeassistant    |   File "/usr/local/lib/python3.8/statistics.py", line 797, in stdev
homeassistant    |     var = variance(data, xbar)
homeassistant    |   File "/usr/local/lib/python3.8/statistics.py", line 739, in variance
homeassistant    |     raise StatisticsError('variance requires at least two data points')
homeassistant    | statistics.StatisticsError: variance requires at least two data points
```

I am not clear, why.

**Comment by dgomes** [comment]
Thanks for reporting! 

I've been testing with 2 datasources, and I believe you are setting up 1 datasource only

I think I've solved the issue in https://github.com/dgomes/rrd_recorder/commit/447ab9516a1ae90bd61861e84ce2ee1970352bc8 (you need to update)

**Comment by ma-zal** [comment]
When I updated code file in my HA instance manually, it started to work. So thanks for the fix!

But when I updated the component by HACS, file stayed old.
I see you have two versions of source code files in GIT. One version is in GIT in root directory (new), second one (older) is in GIT directory `custom_components/rrd/`.

It looks like HACS is taking files from `custom_components/rrd/`, where you have older sourcecode version.
